kubectl_deploy/django/job_migration.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: django-migrations
+spec:
+  template:
+    spec:
+      containers:
+        - name: django
+          image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform-demo-app:latest
+          command: ["/bin/bash","-c","python3 django_reference_app/manage.py makemigrations django_reference_app && python3 django_reference_app/manage.py makemigrations govuk_template_base && python3 django_reference_app/manage.py migrate"]
```

Comment:
Are the `makemigrations` parts necessary here? Usually you'd run `makemigrations` to generate the migration files (like `0001_initial.py`), check those in, then just call `manage.py migrate` in a job like this